homeassistant/components/light/homekit_controller.py

```diff
@@ -0,0 +1,182 @@
+"""
+Support for Homekit lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.homekit/
+"""
+import colorsys
+import json
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_RGB_COLOR, ATTR_COLOR_TEMP, SUPPORT_BRIGHTNESS,
+    SUPPORT_RGB_COLOR, SUPPORT_COLOR_TEMP, Light)
+
+REQUIREMENTS = ['homekit==0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, service):
```

Comment:
Let's keep the method signature the same as other platforms:
 - rename `service` to `discovery_info`
 - default `discovery_info` to `None`

You will also want to return from the method if discovery info is None, in case someone puts the platform config in their config file.

Reply:
Done